cg/meta/report/report_api.py

```diff
@@ -443,6 +446,17 @@ def get_sample_required_fields(case: CaseModel, required_fields: list) -> dict:
 
         return required_sample_fields
 
+    @staticmethod
+    def get_timestamp_required_fields(case: CaseModel, required_fields: list) -> dict:
+        """Retrieves sample timestamps required fields"""
+
+        for sample in case.samples:
+            if sample.application.external:
+                required_fields.remove("received_at")
+                break
```

Comment:
Do we really want to `break` here, or do we want to use `continue`?

Reply:
Thanks for the review @Vince-janv!

Here we just want to remove "received_at" from the `required_fields` constant if any of the case samples is external, so there is no need to keep iterating over it. In fact, `remove()` will throw us an error if we try to remove something that is not in the list anymore

Reply:
I see, so required_fields is on a case level? Then please ignore my comment 😉

Reply:
Its kind of on the report level as this constant is shared between mip and balsamic to validate the data :)